examples/LaserWakefield/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -102,6 +102,27 @@ typedef Particles<ParticleDescription<
 
 /*--------------------------- ions -------------------------------------------*/
 
+/*! Specify (chemical) element
+ * 
+ * Proton and neutron numbers define the chemical element that the ion species
+ * is based on. This value can be non-integer for physical models taking  
+ * charge shielding effects into account. @see http://en.wikipedia.org/wiki/Effective_nuclear_charge
+ * 
+ * It is wrapped into a struct because of C++ restricting floats from being 
+ * template arguments. 
+ * 
+ * Do not forget to set the correct mass of the atom in @see physicalConstants.param !
+ */
+struct NumberOfProtons
+{
+    static const float_X value = 2.0;
+};
+
+struct NumberOfNeutrons
```

Comment:
Is there any future case were we can have only `NumberOfProtons` or `NumberOfNeutrons`?
If not and every time both numbers must be set than it might be better to join both flags to one.

e.g.
```C++
alias coreElements;

struct Helim
{
  static const float_X protons = 1.0;
  static const float_x neutrons = 1.0;
};

//add  `coreElements` to the flag list
bmpl::vector<
  neutronNumber<NumberOfNeutrons>,
  ionizer<particles::ionization::BSI<PIC_Electrons> >
> ParticleFlagsIons;
```